services/importer/lib/importer/runner.rb

```diff
@@ -277,8 +279,42 @@ def single_resource_import
 
               log.store   # Checkpoint-save
               log.append "Filename: #{source_file.fullpath} Size (bytes): #{source_file.size}"
-              import_stats = execute_import(source_file, @downloader)
-              @stats << import_stats
+
+              # Hack for Samples 2.0 Save As - Tech debt to clean up with more elegant solution
+              if(source_file.extension.split('.').last == 'gpkg')
+                md = Carto::GpkgCartoMetadataUtil.new( geopkg_file: source_file.fullpath )
+
+                # Check if FDW
+                if(md.metadata.has_key?('data') and
```

Comment:
Ruby's `and`s are dangerous business. Best to use `&&`

Reply:
Updated in https://github.com/cartodb-org/cartodb/pull/213/commits/fafbaaaf15edcc7e49eb1873b9a7e3d2067c7215